tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -534,6 +534,12 @@ function updateNpmScripts(tree: Tree, options: NormalizedSchema) {
 
     Object.assign(json.scripts, scripts);
 
+    const isNodePackage = options.projectConfig.tags?.some(tag => tag === 'platform:node');
```

Comment:
we already have this function to determine package type, can we use that pls? https://github.com/microsoft/fluentui/blob/master/tools/generators/migrate-converged-pkg/index.ts#L456